minkindr/include/kindr/minimal/implementation/transform-2d-inl.h

```diff
@@ -135,6 +135,13 @@ bool Transformation2DTemplate<Scalar>::operator!=(
   return !(*this == rhs);
 }
 
+template <typename Scalar>
+std::ostream & operator<<(std::ostream & out,
+                          const Transformation2DTemplate<Scalar>& rhs) {
+  out << rhs.asVector().transpose();
```

Comment:
I saw that for 3d, we output the transformation matrix. but I guess you chose the asVector() output on purpose because this is more intuitive, right?

Reply:
Yes, I decided to print as a vector, similar to the [angle axis](https://github.com/ethz-asl/minkindr/blob/master/minkindr/include/kindr/minimal/implementation/angle-axis-inl.h#L257) rather than the 3d version.

Reply:
good

Reply:
I see a risk here that human readers of this output might get the meaning of the components wrong. For an output of a rotation vector as your link points to this much less of a risk because of the canonic order x,y,z. But there is no canonic order between the angle and the position components of a 2d transformation. What about outputting something like `(0, (1, 2))` or `(0 rad, 0, 1)`